backend/src/schema/resolvers/registration.js

```diff
@@ -40,7 +40,7 @@ export default {
       `,
         { nonce, email },
       )
-      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('Email'))
+      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('EmailAddress'))
       if (!emailAddress) throw new UserInputError('Invalid email or nonce')
```

Comment:
what would we have to do to remove `neode` from production code?
This is one of the only remaining resolvers which uses it...

Reply:
The fact that @johnymontana considers using neode in neo4j-graphql-js is making me unsure about if we should try to refactor neode out of our production code.